staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/schemas.go

```diff
@@ -103,17 +148,33 @@ func SchemaDeclType(s *schema.Structural, isResourceRoot bool) *DeclType {
 		}
 		return NewObjectType("object", fields)
 	case "string":
-		if s.ValueValidation != nil {
+		if s.ValueValidation != nil && s.ValueValidation.Format != "" {
```

Comment:
`s.ValueValidation.Format` of  `"" ` is not the only formoat not in the switch statement cases.c I don't see any reason add this check here.

Reply:
Makes sense, I'll remove it.